buildnumber.dat

```diff
@@ -0,0 +1 @@
+3
```

Comment:
Please undo this change. We don't want the build number.dat on the master.

Reply:
Removed